x-pack/plugins/security_solution/public/management/pages/endpoint_hosts/store/middleware.test.ts

```diff
@@ -390,4 +406,64 @@ describe('endpoint list middleware', () => {
       expect(failedAction.error).toBe(apiError);
     });
   });
+
+  describe.only('loads selected endpoint details', () => {
```

Comment:
I assume you forgot to remove this `.only` exclusive here ?

Reply:
oh yes! thank you :D